### ISSUE TITLE
Clear out previously loaded scripts and stylesheets on render.

### ISSUE THREAD

**Issue by samussiah** [RhoInc/CAT#36]

**Comment by jwildfire** [comment]
Is this done in v0.5.1, @samussiah?

**Comment by samussiah** [comment]
Stylesheets were addressed in v0.5.1 by adding the following code to `src/cat/controls/initSubmit.js`:
```
      //Disable and/or remove previously loaded stylesheets.
      d3
        .selectAll("link")
        .filter(function() {
          return !this.href.indexOf("css/cat.css");
        })
        .property("disabled", true)
        .remove();
      d3
        .selectAll("style")
        .property("disabled", true)
        .remove();
```
As I recall scripts aren't an issue because adding a new script to the DOM overwrites any conflicting previously loaded variables.
